CalendarView.m

```diff
@@ -21,6 +23,7 @@ - (NSAttributedString *)attributedStringForObjectValue:(id)anObject withDefaultA
 }
 @end
 
+static int WEEKDAYSTOP = 1;
```

Comment:
A detail but why is this a static instead of a property of CalendarView ?

Reply:
Please fix it, I removed this line at all, but then got compile error so I restored it back (it is from some of the older commits).

Reply:
I'll take a look. In the meantime can you confirm that latest git version builds and runs cleanly ?

Reply:
Yes, it builds and runs ok, I tested running with both `calendarDirection Horizontal` and `calendarDirection Vertical`.
There are 2 warnings though:
```
[nerijus@nerijus simpleagenda-master]$ make
This is gnustep-make 2.9.0. Type 'gmake print-gnustep-make-help' for help.
Running in gnustep-make version 2 strict mode.
Making all for app SimpleAgenda...
 Compiling file AppController.m ...
AppController.m:96:49: warning: 'NSDate' may not respond to 'compare:withTime:'
  return [[[a valueForKey:@"object"] startDate] compare:[[b valueForKey:@"object"] startDate] withTime:YES];
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
AppController.m:316:25: warning: 'NSDate' may not respond to 'compare:withTime:'
  return [[a startDate] compare:[b startDate] withTime:YES];
          ~~~~~~~~~~~~~ ^
2 warnings generated.
```

Reply:
No big deal, thanks